interfaces/HO-Portal/src/app/pages/registration-details/registration-details.page.html

```diff
@@ -56,6 +56,7 @@
             [person]="person"
             [paId]="paId"
             [programId]="programId"
+            [program]="program"
```

Comment:
@agnesesaulite it seems unnecessary to pass both 'program' and 'programId'. So if you need the full 'program' object now, you canremove 'programId' here.

Reply:
I need both of them, because Program comes from Program class (Porgram model) but ProgramId comes from Person class (person model)

Reply:
I don't get this. I just did a commit to explain exactly what I mean. (Which we can revert of course if it turns out I'm wrong.)

Reply:
I added one more commit to this, as programId was still used in the html as well. But I found out that all those parameters that you pass to the accordion component are not actually used there.